main/src/main/scala/sbt/internal/WelcomeBanner.scala

```diff
@@ -0,0 +1,14 @@
+package sbt.internal
+
+private[sbt] object WelcomeBanner {
+  def apply(version: String): String =
+    s"""
+       |Welcome to sbt $version.
+       |Here are some highlights of this release:
+       |  - Coursier: new default library management using https://get-coursier.io
+       |  - Super shell: displays actively running tasks
+       |  - Turbo mode: makes `test` and `run` faster in interactive sessions. Try it by running `set ThisBuild / turbo := true`.
+       |See https://www.lightbend.com/blog/sbt-1.3.0-release for full release notes.
+       |Hide this banner from future sbt sessions by running `skipWelcomeBanner`.
```

Comment:
Eventually most people would run `skipWelcomeBanner`, and then they won't see the banner for sbt 1.4.0.

Reply:
Actually they will see the 1.4.0 file because it writes to a version specific file. We could change the banner for 1.3.1 if we want and have it re-display.

Reply:
Maybe something like?
> Hide the banner for this release by running `skipBanner`.

Reply:
sure. https://github.com/sbt/sbt/pull/4785/commits/5d2c60496f977c87b6f02b523df1052ae6b722f7